### PR TITLE
OCPBUGS-14631: use openshift docs for operatorhub prerequsiste instructions

### PR DESCRIPTION
**Pull request by alebedev87**
The instructions for the STS mode come from the openshift docs now, not upstream.

### CODE REVIEW

File: bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -101,61 +101,68 @@ spec:
...
+      oc get secret -n aws-load-balancer-operator aws-load-balancer-operator
+      ```
 
     After this the operator can be installated through the console or through the command line.
 
-    __Note about UPI:__ For UPI based installs, additional documentation can be found in
-    [VPC and subnets](https://github.com/openshift/aws-load-balancer-operator/blob/main/docs/prerequisites.md#vpc-and-subnets)
+    #### Instructions for STS cluster (inclusing ROSA in STS mode)
+
+    [Bootstrapping AWS Load Balancer Operator on Security Token Service cluster](https://docs.openshift.com/container-platform/4.13/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html#nw-bootstra-albo-on-sts-cluster_albo-sts-cluster).
```
Comment (gcs278) [inline bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml:161]: TIL you can use `latest` in docs link to reference latest version. I feel like that's better than hardcoding 4.13 here. OSSM also uses [latest](https://github.com/maistra/istio-operator/blob/maistra-2.4/manifests-servicemesh/2.4.3/servicemeshoperator.v2.4.3.clusterserviceversion.yaml#L149) in their links. Let me know if you disagree.
```suggestion
    [Bootstrapping AWS Load Balancer Operator on Security Token Service cluster](https://docs.openshift.com/container-platform/latest/networking/aws_load_balancer_operator/installing-albo-sts-cluster.html#nw-bootstra-albo-on-sts-cluster_albo-sts-cluster).
```

**Review by gcs278 — COMMENTED** [review]
Just one comment, everything else looks good.

File: bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (alebedev87) [reply bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml:161]: I agree that `latest` makes sense for `main` branch. We will have to remember to change the link back to the one with "OCP version pinned" when we do a new release (create a new release branch). Just noticed that we already made a similar mistake for the UPI based instructions: [release-0.2 links](https://github.com/openshift/aws-load-balancer-operator/blob/release-0.2/config/manifests/bases/aws-load-balancer-operator.clusterserviceversion.yaml#L73) `main` branch documentation.

Comment (alebedev87) [reply bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml:161]: > Just noticed that we already made a similar mistake for the UPI based instructions: [release-0.2 links](https://github.com/openshift/aws-load-balancer-operator/blob/release-0.2/config/manifests/bases/aws-load-balancer-operator.clusterserviceversion.yaml#L73) main branch documentation.

@gcs278 : can you help me to fix the mistake?

- https://github.com/openshift/aws-load-balancer-operator/pull/100
- https://github.com/openshift/aws-load-balancer-operator/pull/101
- https://github.com/openshift/aws-load-balancer-operator/pull/102

Comment (alebedev87) [reply bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml]: Link updated to use `latest`.

Comment (gcs278) [reply bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml]: > We will have to remember to change the link back to the one with "OCP version pinned" when we do a new release (create a new release branch).

Sure, but I wonder if this process will be forgotten in the future. I wonder if there a better way to do this, like using a version file with mapping of ALBO to OCP releases, and using that version file to generate docs with the right links. Probably overkill for now, just thinking out loud.

Either way, we can proceed with remembering to adjust it later.

**Comment by gcs278** [comment]
@alebedev87 whoops, I'm not in the owner's file. You can approve yourself.

**Comment by alebedev87** [comment]
@gcs278 : indeed, I fixed this with https://github.com/openshift/aws-load-balancer-operator/pull/103.

File: bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (alebedev87) [reply bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml]: > Sure, but I wonder if this process will be forgotten in the future. 

Yes, there is a high chance indeed.

> I wonder if there a better way to do this, like using a version file with mapping of ALBO to OCP releases, and using that version file to generate docs with the right links. Probably overkill for now, just thinking out loud.

We can think of it. This repository misses such a mapping ([similar to ExternalDNS](https://github.com/openshift/external-dns-operator/blob/main/docs/openshift.md#release-branch-mapping)).

**Pull request merged**